library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -931,6 +931,7 @@
     <string name="settings_call_notifications_preferences">Configure Call Notifications</string>
     <string name="settings_silent_notifications_preferences">Configure Silent Notifications</string>
     <string name="settings_system_preferences_summary">Choose LED color, vibration, sound…</string>
+    <string name="settings_error_toggle_pusher">Something went wrong. Please check your network connection and try again.</string>
```

Comment:
Maybe use a more generic key for this string, which could be re-use in other circumstances.
Something like `error_check_network` (we already have `error_no_network`)